src/flux-capacitor.ts

```diff
@@ -100,9 +100,7 @@ class FluxCapacitor extends EventEmitter {
   }
 
   detailsWithRouting(product: Store.Product | Record) {
-    return this.config.details.alwaysFetch
-      ? this.store.dispatch(this.actions.fetchProductDetails(SearchAdapter.extractAllMeta(product).id))
-      : this.store.dispatch(this.actions.setDetails(product));
+    this.store.dispatch(this.actions.setDetails(product));
   }
```

Comment:
remove `alwaysFetch` from configuration bc details should always always fetch now